src/test/lido/ObolLidoSplit.t.sol

```diff
@@ -158,16 +158,17 @@ contract ObolLidoSplitTest is ObolLidoSplitTestHelper, Test {
 
   function testFuzz_CanDistributeWithFee(
     address anotherSplit,
-    uint256 amountToDistribute,
+    uint8 amountToDistributeEth,
```

Comment:
fuzzing was failing for this test reliably, hence reduced the ranges.

Reply:
This is in ETH. We multiply it below `uint256(amountToDistributeEth) * 1 ether;`.